src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -104,6 +105,7 @@ public static Task<RazorLanguageServer> CreateAsync(Stream input, Stream output,
                     .WithHandler<CodeActionResolutionEndpoint>()
                     .WithHandler<MonitorProjectConfigurationFilePathEndpoint>()
                     .WithHandler<RazorComponentRenameEndpoint>()
+                    .WithHandler<RazorDefinitionEndpoint>()
```

Comment:
Hmm, not something we need to solve in this PR but I feel like as we expand our capabilities this `WithHandler` section is going to get unruly fast, I wonder if we could set up some way that the various endpoints self-register. Or maybe this is a pointless concern and I'm just picky.

Reply:
I would worry about further reflection/attribute usage. That can also get pretty crazy :0